syntax = "proto3";

option go_package = "pb/game";
import "common/comp.proto";

enum eCombatState
{
	kSilence = 0;
}

message CombatStatePbComponent 
{
    map<uint64, bool> sources = 1;
}

message CombatStateCollectionPbComponent 
{
    map<uint32, CombatStatePbComponent> state_collection = 1;
}
